### PR TITLE
Prevents tourettes from stacking

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
:cl:
tweak: tourettes doesnt stack on itself anymore
/:cl:
this shit was cancer it would keep rolling for stun while stunned so you had like 2-3 whole minutes of stun

### CODE REVIEW

**Comment by XDTM** [comment]
should be marked balance, but i approve of this change

periodic stuns in genetics are already covered by seizures

**Comment by XDTM** [comment]
to fix that you'd add an IsStun() check on the mob, this is by all means a balance change instead

**Comment by CitrusGender** [comment]
eh, good PR but this is definitely balance.

Genetics really has no risk anymore though other than accidentally using a monkey SE

**Comment by XDTM** [comment]
No longer balance, it's been changed and now only checks if you're already stunned

**Review by SpaceManiac — DISMISSED** [review]

**Comment by Tlaltecuhtli** [comment]
why balance tag @SpaceManiac

**Comment by Cyberboss** [comment]
Everyone else already said it, this isn't a fix, you just reduced the stun times

**Comment by Tlaltecuhtli** [comment]
i reverted and set the same stun time why does no one read the commits

**Comment by ExcessiveUseOfCobblestone** [comment]
It's hard to translate "dsdsvfsf" and "fddasdc"...

**Comment by Tlaltecuhtli** [comment]
http://take.ms/j4IGK

File: code/datums/mutations/body.dm [hunk 1]
```diff
@@ -102,7 +102,8 @@
 
 /datum/mutation/human/tourettes/on_life(mob/living/carbon/human/owner)
 	if(prob(10) && owner.stat == CONSCIOUS)
-		owner.Stun(200)
+		if(!owner.IsStun())
```
Comment (ShizCalev) [inline code/datums/mutations/body.dm]: put this at the end of 104

**Review by ShizCalev — DISMISSED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
Is there anywhere that allows for stun stacking from the same source?

Just curious why we don't have an argument for the source of the stun within the stun proc to avoid stacking automatically vs having bits of isstunned in multiple locations.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**